src/uu/yes/Cargo.toml

```diff
@@ -16,11 +16,10 @@ path = "src/yes.rs"
 
 [dependencies]
 clap = { workspace=true }
-libc = { workspace=true }
 uucore = { workspace=true, features=["pipes"] }
 
-[target.'cfg(any(target_os = "linux", target_os = "android"))'.dependencies]
-nix = { workspace=true }
+[target.'cfg(any(target_os = "linux", target_os = "android",unix))'.dependencies]
```

Comment:
Shouldn't this just be `cfg(unix)`? Android & Linux are subsets of the unix cfg right?

Reply:
@tertsdiepraam this was what I had done in  b60f745. 

I have done a mistake it seems, reverting it rn.